spec/services/case_filter/open_case_status_filter_spec.rb

```diff
@@ -22,6 +22,7 @@
       std_responded_foi
       trig_responded_foi
       full_responded_foi
+      std_rejected_sar
```

Comment:
Not sure whether this is the correct naming convention. Standard workflow, rejected offender sar case. Followed the comments in `spec/support/standard_setup.rb`